modules/clearUi.js

```diff
@@ -0,0 +1,8 @@
+/* eslint-disable*/
+const input = document.querySelector('.input');
+
+const clear = () => {
+  input.value = '';
```

Comment:
- [ ] [REQUIRED] This file `clearUi.js` should contain methods related to the status updates (`completed`: `true` / `false`).
- [ ] It has an event listener on the checkbox (`change`). 
- [ ] This file should contain the function for the "Clear all completed" button using the `Array.filter()` method.